src/ContextFreeTasks.NETStandard/ContextFreeTasks.NETStandard.csproj

```diff
@@ -6,7 +6,11 @@
     <Compile Include="**\*.cs" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="NETStandard.Library" Version="1.6.1" />
+    <PackageReference Include="System.Resources.ResourceManager" Version="4.3.0">
```

Comment:
yes, netcoreapp application and netstandard library project are based on NETStandard.Library package.
but some other platforms (targeting net, mono, or etc, used like PCL) aren't dependent to NETStandard.Library.
if use package depend to NETStandard.Library, ALL OF NetStandard packages are installed, and almost all are not used.